gslab_scons/provenance.py

```diff
@@ -129,8 +132,13 @@ def write_readme(readme_path, provenance_path):
     '''
     Writes readme to provenance.
     '''
-    with open(readme_path, 'rU') as f:
-        out = '%s' % f.read()
+    try:
+        with open(readme_path, 'rU') as f:
+            out = '%s' % f.read()
+    except IOError:
+        raise IOError(('%s does not exist.\n If you are in release mode,' % readme_path) +
```

Comment:
Error message isn't true. The same exception will be raised if you try to read a directory.

Reply:
We can use `sys.argv[0]` to determine the name of the script that's being executed. We should check if that value ends in `release.py` and only print the release-related stuff if it does.

Reply:
I think that's overly complicating things. This kind of message suffice, you aren't constantly running `provenance.py` anyway.

Reply:
Won't we be directly running provenance.py for raw data on Dropbox?

Reply:
Yup.